Makefile

```diff
@@ -11,7 +11,7 @@ lib: libPCM.a
 
 klocwork: $(EXE)
 
-CXXFLAGS += -Wall -g -O3 -Wno-unknown-pragmas
+CXXFLAGS += -std=c++11 -Wall -g -O3 -Wno-unknown-pragmas
```

Comment:
the -std=c++11 is added to CXXFLAGS below. Any reasons why you have to add it here as well?